raster/src/main/scala/geotrellis/raster/Implicits.scala

```diff
@@ -60,6 +59,8 @@ trait Implicits
   implicit class withMultibandTileMethods(val self: MultibandTile) extends MethodExtensions[MultibandTile]
       with DelayedConversionMultibandTileMethods
 
+  import geotrellis.vector.methods.Implicits.withExtraPointMethods
```

Comment:
Hmm, feel like avery ad-hock solution. What are the steps to reproduce the issue you had?

Reply:
Is this issue present in one of your PRs with the Scala 2.13 compilation?

Reply:
I see the issue in IntelliJ syntax highlighting by just loading geotrellis master using 2.12 -- the 3 mentioned `Implicits` objects show as red, I assume b/c it's trying to resolve them from the `geotrellis.vector._` import rather than the package they're in, `geotrellis.raster`.

changing the `Implicits` object references from relative to absolute (by just prefixing them with `geotrellis.raster.`) also fixed the issue, but I thought explicitly importing only the one implicit class form `geotrellis.vector` was a better solution.  Why do you say it's ad-hoc?

This problem was the only issue I hadn't resolved in the 2.13 compilation PR.